fp/_baseConvert.js

```diff
@@ -279,6 +279,9 @@ function baseConvert(util, name, func, options) {
       setPlaceholder = true;
       func.placeholder = result.placeholder = placeholder;
     }
+    result.convert = function(options) {
+      return baseConvert(util, name, func, options);
```

Comment:
A test for this could be something like `mapValues.convert({cap:false}).convert({ rearg: false})`.